ndk-build/src/apk.rs

```diff
@@ -32,7 +32,7 @@ impl ApkConfig {
             .join(format!("{}-unaligned.apk", self.apk_name))
     }
 
-    fn apk(&self) -> PathBuf {
+    pub fn apk(&self) -> PathBuf {
```

Comment:
Would be nice to add a doc comment here, and mention this and `DEFAULT_DEV_PASSWORD` in the changelog for `ndk-build` :)